src/butil/iobuf.cpp

```diff
@@ -341,7 +214,7 @@ uint32_t block_size(IOBuf::Block const* b) {
     return b->size;
 }
 
-inline IOBuf::Block* create_block(const size_t block_size) {
+IOBuf::Block* create_block(const size_t block_size) {
```

Comment:
remove the inline keyword may affect performance, you can move the function body to iobuf_inl.h, since that file is for inline functions.